src/main/java/project/bot/service/TenderParserServiceBot.java

```diff
@@ -38,59 +36,43 @@ public void onUpdateReceived(Update update) {
         String userName = update.getMessage().getChat().getUserName();
         String messageText = update.getMessage().getText();
         long chatId = update.getMessage().getChatId();
-        String errorOccurred = "Error occurred: ";
 
         if (!update.hasMessage() || !update.getMessage().hasText()) {
             log.warn("Unexpected update from user " + userName);
-            try {
-                execute(messageSender.sendUnsupportedDataWarning(chatId));
-                log.info("Warn message was sent to user " + userName);
-            } catch (TelegramApiException e) {
-                log.error(errorOccurred + e);
-            }
+            sendMessageToUser(messageSender.sendUnsupportedDataWarning(chatId));
+            log.info("Warn message was sent to user " + userName);
         } else {
             switch (messageText) {
                 case "/start" -> {
-                    try {
-                        execute(messageSender.sendWelcomeMessage(chatId));
-                        log.info("Welcome message was sent to user " + userName);
-                    } catch (TelegramApiException e) {
-                        log.error(errorOccurred + e);
-                    }
+                    sendMessageToUser(messageSender.sendWelcomeMessage(chatId));
+                    log.info("Welcome message was sent to user " + userName);
                 }
                 case "My subscription list" -> {
-                    try {
-                        execute(messageSender.sendUserSubscriptionsList(chatId));
-                        log.info("Subscription list was sent to user " + userName);
-                    } catch (TelegramApiException e) {
-                        log.error(errorOccurred + e);
-                    }
+                    sendMessageToUser(messageSender.sendUserSubscriptionsList(chatId));
+                    log.info("Subscription list was sent to user " + userName);
+
                 }
                 case "Available tender sites for subscription" -> {
```

Comment:
I don't want to type so long message(((
I prefere short ones.

But constats' names can be long)

Reply:
you do not need to type it, this case works when you push such button on the keyboard

Reply:
So why not enums ?